src/Peachpie.Library/DateTime/DateInterval.cs

```diff
@@ -217,7 +230,6 @@ private protected void Initialize(DateInfo ts, bool negative)
         private protected void Initialize(TimeSpan ts)
         {
             _span = ts.Duration(); // absolutize the range
-            days = (int)ts.TotalDays;
```

Comment:
just checking - is it correct to not initialize "days" property here?

Reply:
Yes, because otherwise the following test would fail:
```php
    $interval = new \DateInterval('P2D');
    echo print_r($interval->format('%r%a'), true).PHP_EOL; // <-- should return (unknown)
```